application/migrations/005_create_logging_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+defined('BASEPATH') OR exit('No direct script access allowed');
+
+class Migration_create_logging_table extends CI_Migration
+{
+
+    public function up()
+    {
+        // Create new table for capturing SQL query logs
+        $this->db->query('
+        CREATE TABLE IF NOT EXISTS ci_logs (
```

Comment:
Sorry, I lost track of this...

Following up on https://github.com/GSA/project-open-data-dashboard/pull/218#discussion_r352996958

I'm not sure why CILogger exists, maybe to meet an audit control. The System Security Plan does not refer to it explicitly but refers to a future implementation of a CI plugin that will audit user actions.

If it is for auditing, we should log to file (or stdout), the same way we meet the control for other applications. Putting  that logging information into the DB requires us to come up with a unique solution for rotating and querying that information. Let's not re-invent the wheel.

Let's leave this as is and create an issue. We probably just want to update CILogger to write to file instead of write to DB.

Reply:
Auditing user actions is way different from logging all the SQL that was generated by those actions, and probably better implemented via [`pre-controller` and `post-controller` hooks](https://codeigniter.com/user_guide/general/hooks.html#hook-points). So, I'm guessing we'll rip this out. I'll leave a pointer back to this discussion in [the existing issue](https://github.com/gsa/datagov-deploy/issues/535), which I'm repurposing.